trinity/sync/light/service.py

```diff
@@ -346,7 +346,8 @@ def callback(r: protocol._DecodedMsgType) -> None:
         """
         self.logger.debug("Fetching header %s from %s", encode_hex(block_hash), peer)
         max_headers = 1
-        headers = await peer.requests.get_block_headers(
+
+        headers = await peer.requests.get_block_headers(  # type: ignore
```

Comment:
Same here, why the ignore?  add a comment?

Reply:
I had spend some minutes trying to figure out what is going on but failed. I'll put a comment on it with a hint and todo to follow up on it.

Reply:
Found it:

https://github.com/ethereum/py-evm/blob/master/trinity/protocol/eth/handlers.py#L22-L24

Missing type hint there.

Reply:
mypy already inferred the types of them correctly but we had put an `int` where we should have put a `BlockIdentifier`.

Fixed it with this: https://github.com/ethereum/py-evm/pull/1293/files#diff-0633dd2f497dbac292e7b3dc13094a4aR61